examples/activity/activity.go

```diff
@@ -42,15 +42,15 @@ func (a *Activity) Eval(ctx activity.Context) (done bool, err error) {
 	input := &Input{}
 	err = ctx.GetInputObject(input)
 	if err != nil {
-		return false, err
+		return true, err
```

Comment:
Why was this changed from false to true?

Reply:
Wouldn't we want to exit the activity when we encounter error?

Reply:
If there is an error, that first parameter gets ignored, so it's returning the go default value for a boolean essentially.  Kind of like when you return nil, err in a lot of cases

Reply:
Alright. In the old code it had to be true as far as I know and that's why I changed it to true.